lib/puppet_x/puppetlabs/scheduled_task/trigger.rb

```diff
@@ -62,17 +64,20 @@ def string_to_int(value)
   end
   module_function :string_to_int
 
-  def string_to_date(value)
+  # TODO: rename this to imply better how its parsed?
+  def string_to_date(value, convert_to = :utc)
```

Comment:
@michaeltlombardi any thoughts about this always returning UTC? i.e. rename it to `iso8601_datetime_to_utc` and remove the `convert_to` param? Don't think we need the param for tests... and its really the only code path.

Reply:
Yeah, this makes sense to me.  It's a bit more descriptive, too.